lib/org/chromium/webidl/ast/Implements.js

```diff
@@ -8,6 +8,8 @@ foam.CLASS({
   name: 'Implements',
   extends: 'org.chromium.webidl.ast.DefinitionData',
 
+  ids: [ 'implementer', 'implemented' ],
```

Comment:
This is fine for now. Add a TODO that poses the question of whether we should have a custom id property that delegates to implementer and implemented's getName()